test/metabase/api/collection_test.clj

```diff
@@ -486,63 +519,63 @@
       (mt/with-temp Collection [collection {:name "Debt Collection"}]
         (perms/grant-collection-read-permissions! (perms-group/all-users) collection)
         (with-some-children-of-collection collection
-          (is (= (-> (mapv default-item [{:name "Acme Products", :model "pulse", :entity_id true}
-                                         {:name "Birthday Card", :description nil, :model "card",
-                                          :collection_preview false, :display "table", :entity_id true}
-                                         {:name "Dine & Dashboard", :description nil, :model "dashboard", :entity_id true}
-                                         {:name "Electro-Magnetic Pulse", :model "pulse", :entity_id true}])
-                     (assoc-in [1 :fully_parametrized] true))
-                 (mt/boolean-ids-and-timestamps
-                  (:data (mt/user-http-request :rasta :get 200 (str "collection/" (u/the-id collection) "/items"))))))))
+                                          (is (= (-> (mapv default-item [{:name "Acme Products", :model "pulse", :entity_id true}
```

Comment:
pretty sure the previous indentation is right.
The `with-some-children-of-collection` has a `style/indent=1`

Reply:
fixed